neo/Settings.py

```diff
@@ -32,6 +37,52 @@
 FILENAME_SETTINGS_COZNET = os.path.join(DIR_PROJECT_ROOT, 'protocol.coz.json')
 
 
+class PrivnetConnectionError(Exception):
+    pass
+
+
+class DependencyError(Exception):
+    pass
+
+
+def check_privatenet():
```

Comment:
also, i moved check_privatenet from prompt.py here to settings.

Reply:
This is nitpicky, but does this more appropriately belong in the `SettingsHolder` class? It seems like that may be a more appropriate OO-design, and then you don't need to reference `settings` and instead can just get those directly from `self`. Without this, you could technically have multiple `SettingsHolder` instances with different `RPC_LIST` and `LEVELDB_PATH` values and this would not work as expected.

Reply:
done